cmd/get/puppet.go

```diff
@@ -0,0 +1,22 @@
+package get
+
+import (
+	"github.com/puppetlabs/prm/pkg/prm"
+	"github.com/rs/zerolog/log"
+	"github.com/spf13/cobra"
+)
+
+func createGetPuppetCommand() *cobra.Command {
+	tmp := &cobra.Command{
+		Use:   "puppet",
+		Short: "Gets the Puppet runtime version currently configured",
+		Long:  "Gets the Puppet runtime version currently configured",
+		Run:   getPuppetVersion,
+	}
+
+	return tmp
+}
+
+func getPuppetVersion(cmd *cobra.Command, args []string) {
```

Comment:
I suppose eventually this function will return slightly more info but for now just telling the user what was said is good. 👍🏻

Reply:
If you feel this is a bit lacking, let me know - if I missed something, I should address it.

Reply:
I was thinking about the json output functionality which I realize now we hadn't been capturing per-ticket in our acceptance criteria. If we're going to circle back and implement that in its own ticket, I think this is fine and I'm happy to merge.

Reply:
Ah, OK. Good catch indeed. Let's chat about that and see what the team consensus is

Reply:
Follow up conversation will be happening on this to clarify expectations and a separate ticket will be there to implement the work.